src/main/java/com/ucm/ms/accounts/controllers/CreditCardController.java

```diff
@@ -37,4 +67,119 @@ public CreditCardController(CreditCardSearch creditCardSearch) {
 			return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
 		}
 	}
-}
+	
+	
+		
+		
+		@GetMapping(path = "get_credit_cards")
```

Comment:
The one we already have is from before and it does not have type AccountType. Its of type CreditCard. So I guess we can take down that old method and use this one instead since this is updated one.

Reply:
Oh no that's not an old one, I made that method after we made the database changes. I think there might be some miscommunication here; are these meant to be returning different types of credit cards? As in like different account types that are of type credit card? The method that I made gets every individual credit card in the database. I'm thinking this one has a different purpose, so we should probably rename it.

Reply:
Oh yeah thats prolly it, I will rename it